src/Microsoft.VisualStudio.Web.CodeGeneration.Contracts/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.csproj

```diff
@@ -10,6 +10,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonSoftVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetLegacyVersion)" PrivateAssets="All" />
```

Comment:
Both the CLI tool package and the _inside man_ package have a dependency on this. 
So have to keep the Newtonsoft.json dependency at 9.0.1 here, so instead of letting the package have a reference to old version, marked it as private. The CLI tool and the inside man both provide their respective versions of the package.